modules/configuration/src/main/java/org/apache/ignite/internal/configuration/asm/InnerNodeAsmGenerator.java

```diff
@@ -696,6 +700,16 @@ private List<MethodDefinition> addNodeChangeMethod(
 
         BytecodeBlock bytecodeBlock = new BytecodeBlock();
 
+        if (classDef == innerNodeClassDef) {
+            bytecodeBlock.append(changeMtd.getThis().invoke(ASSERT_MUTABILITY_MTD));
```

Comment:
Show an example of how it will look in byte code.

Reply:
Really? Ok, I will

Reply:
Thanks